doc/manual.rst

```diff
@@ -6,232 +6,299 @@
+To get the best performance out of the evaluated models, *auto-sklearn* uses ensemble selection by `Caruana et al. (2004) <https://dl.acm.org/doi/pdf/10.1145/1015330.1015432>`_
+to build an ensemble based on the models’ prediction for the validation set.
 
-.. _inspecting_the_results:
+.. collapse:: <b>Configure the ensemble building process</b>
+
+    The following hyperparameters control how the ensemble is constructed:
+
+    * ``ensemble_size`` determines the maximal size of the ensemble. If it is set to zero, no ensemble will be constructed.
+    * ``ensemble_nbest`` allows the user to directly specify the number of models considered for the ensemble.  This hyperparameter can be an integer *n*, such that only the best *n* models are used in the final ensemble. If a float between 0.0 and 1.0 is provided, ``ensemble_nbest`` would be interpreted as a fraction suggesting the percentage of models to use in the ensemble building process (namely, if ensemble_nbest is a float, library pruning is implemented as described in `Caruana et al. (2006) <https://dl.acm.org/doi/10.1109/ICDM.2006.76>`_).
+    * ``max_models_on_disc`` defines the maximum number of models that are kept on the disc, as a mechanism to control the amount of disc space consumed by *auto-sklearn*. Throughout the automl process, different individual models are optimized, and their predictions (and other metadata) is stored on disc. The user can set the upper bound on how many models are acceptable to keep on disc, yet this variable takes priority in the definition of the number of models used by the ensemble builder (that is, the minimum of ``ensemble_size``, ``ensemble_nbest`` and ``max_models_on_disc`` determines the maximal amount of models used in the ensemble). If set to None, this feature is disabled.
+
+.. collapse:: <b>Inspect the final ensemble</b>
+
+    The results obtained from the final ensemble can be printed by calling ``show_models()``.
+    The *auto-sklearn* ensemble is composed of scikit-learn models that can be inspected as exemplified
+    in the Example :ref:`sphx_glr_examples_40_advanced_example_get_pipeline_components.py`.
+
+.. collapse:: <b>Fit ensemble post-hoc</b>
+
+    To use a single core only, it is possible to build ensembles post-hoc. An example on how to do this (first searching
```

Comment:
Same thing, not sure why the single core here is relevant to ensemble building.

Reply:
Agreed, this is not relevant (and also outdated). Fixed.